### PR TITLE
Tutorial

### PR DESCRIPTION
**Pull request by dominikkawka**

### CODE REVIEW

**Comment by laurafitzgerald** [comment]
@dominikkawka can you put the changes into a file other than the README. You could put it into something like tutorial.md

File: Tutorial.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
++Before you start the tutorial, we will assume that you have git installed onto your computer.
+-test	+If you don’t have git, then you can learn how to install it [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Here is a guide on the commands of [git also.](http://feedhenry.org/student-help-guide/topic04-git-introduction/book-git-tutorial/index.html#/Basics)
```
Comment (laurafitzgerald) [inline Tutorial.md]: I think this `-test` is a copy paste error?

File: Tutorial.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+-K.H
```
Comment (laurafitzgerald) [inline Tutorial.md]: same here, copy and paste error?

File: Tutorial.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
++Before you start the tutorial, we will assume that you have git installed onto your computer.
+-test	+If you don’t have git, then you can learn how to install it [here](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git). Here is a guide on the commands of [git also.](http://feedhenry.org/student-help-guide/topic04-git-introduction/book-git-tutorial/index.html#/Basics)
+-K.H	
+ 	 
+-This is a screenshot for a tutorial	+Before you can add your own changes to the school website, you first need to “fork” the repo of which that you want to change. To “fork” a repo means to get your own copy of the repo, in this case our school website.  
+-:)	
+ 	 
++![cap1](https://i.imgur.com/rzHlCWQ.png)
```
Comment (laurafitzgerald) [inline Tutorial.md]: Will these images remain in this location long term?

File: Tutorial.md (hunk 1 above)
Comment (dominikkawka) [reply Tutorial.md]: oops

File: Tutorial.md (hunk 3 above)
Comment (dominikkawka) [reply Tutorial.md]: the images will remain there as long as imgur stays up, should I push them to the repo?

**Comment by dominikkawka** [comment]
I don't understand what you mean by "you link to it from your contributing.md once that is created".  I didn't make an contributing.md file, should I rename tutorial.md to contributing.md?

**Comment by ciaranRoche** [comment]
@dominikkawka the tutorial is looking good, I would recommend creating a `docs` folder in the repo, you could put your git tutorial in there it will save the repo from getting cluttered as docs/tutorials get added, the usual convention on a git repo would be to have the `CONTRIBUTING.md` and `README.md` in the source folder so they can be easily accessed.
What Laura means by linking to the `CONTRIBUTING.md` is a hyper link that people can click on to view your tutorial page. You can specify it in the `CONTRIBUTING.md` something like `First clone the repo, a tutorial on how to do this can be viewed here` where `here` would be clickable and bring people to your tutorial page.

**Comment by dominikkawka** [comment]
Having problems how to use git bash on the school pcs, I'm going to make the docs file at home

**Comment by laurafitzgerald** [comment]
@AdamLalor same here, can you leave a comment before merging a pr. There was some comments/suggestions made by @ciaranRoche that weren't resolved before this was merged. As a general rule of thumb the change should be approved by 2 other team members and a comment to that effect left in the pr before anything is merged.

**Pull request merged**